packages/larreco/package.py

```diff
@@ -50,6 +48,8 @@ class Larreco(CMakePackage, FnalGithubPackage):
     depends_on("rstartree")
     depends_on("tbb")
 
+    patch('09.25.00.patch', when='@09.25.00')
```

Comment:
Are you able to make a `larreco` PR with the changes in this patch?  If not, let me know and I will do it.